src/d3d11/d3d11_texture.cpp

```diff
@@ -730,33 +730,34 @@ namespace dxvk {
   
   VkImageLayout D3D11CommonTexture::OptimizeLayout(VkImageUsageFlags Usage) {
```

Comment:
Have you considered moving this function to the backend? It's used in both the d3d9 and d3d11 frontend and only uses the Vulkan usage as parameter.

Reply:
I have, but for now I think it's more useful to keep it here since we don't *always* want to change the default layout when adding usage flags, and the backend doesn't really know how the front-end intends to actually use the image.